packages/sdk-drop-in/tests/iframe.test.ts

```diff
@@ -0,0 +1,93 @@
+import { test, expect } from '@playwright/test';
+
+import * as process from 'node:process';
+
+import { WidgetType } from '../src/apps/MoniteApp';
+
+const consumerPage = '/monite-iframe-app-demo';
+
+const routingPaths: Record<WidgetType, string> = {
+  payables: '/payables',
+  receivables: '/receivables',
+  counterparts: '/counterparts',
+  products: '/products',
+  tags: '/tags',
+  'approval-policies': '/approval-policies',
+  onboarding: '/onboarding',
+};
+
+test.describe('Monite Iframe Integration', () => {
+  test.beforeEach(async ({ page }) => {
+    await page.route('/config.json', async (route) => {
+      if (process.env.CI) {
+        await route.fulfill({
+          contentType: 'application/json',
+          body: process.env.MONITE_E2E_APP_ADMIN_CONFIG_JSON,
+        });
+      } else {
+        await route.continue();
+      }
+    });
+
+    await page.goto(`${consumerPage}${routingPaths.receivables}`);
+  });
+
+  test('test the theme switcher', async ({ page }) => {
+    // log the html string body what is on page for testing
+    await page.waitForTimeout(5000);
+    const body = await page.locator('body').innerHTML();
+    console.log(body); //ToDo: see the html string body what is on page for testing
+
+    await page.getByRole('button', { name: 'Material UI' }).click();
+    await page.getByText('Theme').click();
+    await page.getByRole('button', { name: 'Material UI' }).click();
+    await page.getByRole('menuitem', { name: 'Monite' }).click();
+    await page.getByRole('button', { name: 'Monite' }).click();
+    await page.getByLabel('Dark Mode').check();
+    await page.locator('.MuiBackdrop-root').click();
+
+    const iframe = page.frameLocator('iframe');
+    await iframe.locator('body').waitFor({ state: 'visible' });
+
+    const themeElement = iframe.locator('body');
+    const bgColor = await themeElement.evaluate(
+      (el) => getComputedStyle(el).backgroundColor
+    );
+
+    expect(bgColor).toBe('rgb(18, 18, 18)');
+  });
+
+  test('test the Roles button under Settings', async ({ page }) => {
+    await page.getByRole('button', { name: 'Settings' }).click();
+    await expect(page.getByRole('button', { name: 'Roles' })).toBeVisible();
+    await page.getByRole('button', { name: 'Roles' }).click();
```

Comment:
Could you also add a check to make sure the component(iframe) is actually loaded?
```ts
await expect(iframe.getByRole('heading', { name })).toBeVisible();
```
It would be great to test items working in this way too, the same as you did with `should see the ${name.toLowerCase()} tab`